model/flow/order/identity.go

```diff
@@ -11,17 +11,3 @@ import (
 func ByNodeIDAsc(identity1 *flow.Identity, identity2 *flow.Identity) bool {
 	return bytes.Compare(identity1.NodeID[:], identity2.NodeID[:]) < 0
 }
-
-func ByReferenceOrder(nodeIDs []flow.Identifier) func(*flow.Identity, *flow.Identity) bool {
```

Comment:
I would suggest _not_ removing this method. It might be useful in the future (or does the Go-compiler complain?)

Reply:
The linter might complain, I'll try to put it back 👌

Reply:
All good!